app.py

```diff
@@ -222,19 +223,16 @@ def list_data_bundles():
         res = dss.post_search._request(dict(replica='aws', per_page=per_page, es_query={}))
     # We need to page using the github style
     if res.links.get('next', None):
-        try:
-            # first search_after item of the query string in the link
-            # header of the response
-            next_page_token = urlparse.parse_qs(
-                urlparse.urlparse(
-                    res.links['next']['url']).query)['search_after'][0]
-        except Exception as e:
-            print(e)
+        # first search_after item of the query string in the link
+        # header of the response
+        next_page_token = urlparse.parse_qs(
+            urlparse.urlparse(
+                res.links['next']['url']).query)['search_after'][0]
     # And convert the fqid message into a DOS id and version
     response = {}
     response['next_page_token'] = next_page_token
     try:
-        response['data_bundles'] = map(dss_list_bundle_to_dos, res.json()['results'])
+        response['data_bundles'] = list(map(dss_list_bundle_to_dos, res.json()['results']))
```

Comment:
Is the `list` command needed for Python 3? How do you ensure Python 2 compatibility? Or would that raise an exception?

Reply:
Hi Natan, thanks for taking the time to answer my comments / questions. In know you mentioned your work and actions during standup, but I see commits and PRs more as a form of record keeping so we can revisit it later and know what was done and why.